src/main/java/kitchenpos/domain/MenuGroup.java

```diff
@@ -1,22 +1,32 @@
 package kitchenpos.domain;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
 public class MenuGroup {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+
     private String name;
 
-    public Long getId() {
-        return id;
+    protected MenuGroup() {
     }
 
-    public void setId(final Long id) {
-        this.id = id;
+    public MenuGroup(final String name) {
+        this.id = null;
+        this.name = name;
     }
```

Comment:
```suggestion
    public MenuGroup(final String name) {
        this.name = name;
    }
```

여기도 다른 도메인 객체들처럼 생성자에서 id에 null을 명시적으로 넣어주지 않아도 된다고 생각해요.
어떻게 생각하시나요??

Reply:
Long의 디폴트가 null이니 명시적으로 안넣어도 될것같네요!

수정했습니다!